packages/manager/apps/web/client/app/hosting/hosting.service.js

```diff
@@ -380,6 +380,41 @@ import union from 'lodash/union';
         });
       }
 
+      /**
+       * Retrieve e-mail options
+       * @param {string} serviceName
+       */
+      getEmailOptions(serviceName) {
+        return this.OvhHttp.get(`/hosting/web/${serviceName}/emailOption`, {
```

Comment:
Not already in `ovh-api-services` ?

Reply:
This legacy service is already using OvhHttp, i am wondering if it make sense to inject ovh-api-service. It's not as if ovh-api-services is providing any benefits ... I bet that we'll probably kill ovh-api-services someday. I prefer to keep using OvhHttp for consistency but if the majority of you wants me do do the change then i guess i'll have to! 🤖